ion_hash_tests.ion

```diff
@@ -1063,6 +1063,63 @@ structCount14::{
     final_digest::(0x0b 0xd0 0x0c 0x0b 0x70 0x66 0x6c 0x6f 0x61 0x74 0x5f 0x7a 0x65 0x72 0x6f 0x0c 0x0e 0x0c 0x0b 0x40 0x0c 0x0e 0x0e)
   ) }
 }
+
+// This is a modified version of the failing struct test below (replaced symbol: ms [update bytes: 0x6d, 0x73] to int: 0 [update bytes: 0x0b,0x20, 0x0e])
```

Comment:
Are we sure the original test failed due to the symboil `ms`? I thought the symbol's limitations was for SIDs like `$1` or `$42` but we could do non-SID symbols like `ms` or `hello` (which is earlier in the file).

Reply:
Sorry, for the late update. I was debugging against dotnet and the update bytes were mismatching at symbol: ms. The hashReader.value() was returning a symbol table for a symbolType. The other symbol tests were passing because they only had one symbol in the symbol table and they were calling symboltable[0] within writer.writeSymbol(). This test has multiple symbols so it is not writing the current symbol.

I updated hashReader.value() to return the string value and I am getting the correct update bytes but there are more byte mismatch which I am investigating

Reply:
Fixed two bugs that was causing this test to fail:

1. Updated hashReader.Value() to return symbol text instead of symbol table
2. Updated CryptoHasher.Sum() to reset hash after calling hashAlgorithm.Sum()